test/api/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
extra newline?

Reply:
Removed extra lines from both `test_custom_op.cpp` and `CMakeLists.txt`.